GiniVisionExample/GiniVisionExample/AppHelper.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  AppHelper.swift
+//  GiniVisionExample
+//
+//  Created by Enrique del Pozo Gómez on 11/22/17.
+//  Copyright © 2017 Gini. All rights reserved.
+//
+
+import Foundation
+import GiniVision
+
+final class AppHelper {
```

Comment:
I don't really like the name? App helper? Does it help the app... how?

Reply:
I'm very bad naming this kind of classes, do you have a better idea?

Reply:
Idk, `AppVersion`?